### DISCUSSION TITLE
Typing raw query in simple case, `select * from table where (... complicated raw condition)`?

### DISCUSSION THREAD

**Discussion by xinluh** [prisma/prisma#17382] [Q&A] [answered]
We have a bunch of queries that have rather complicated `where` clause (sometimes involving complicated joins) but simple `select *` as output. Is there way to do raw query but still benefit from the regular parsing/typing etc. of typed ORM?

i.e. almost like 
```
prisma.someModel.findMany({ where: Prisma.sql`...`})
```

**Comment by nurul3101** [comment, marked as answer]
Hi @xinluh 👋 

That's not possible at the moment. However, we have a feature request for it here:
- https://github.com/prisma/prisma/issues/5560

If you can add a 👍 to the feature request, then it would help our product team in prioritising it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
